configs/openmrs_config/privileges/privileges.csv

```diff
@@ -17,4 +17,6 @@ dc023a57-1bda-4e7e-8c71-d71539a18efc,View NCD Screening encounters,Able to view
 8756636b-de4a-4d7b-a41f-ed10787c4c73,View NCD Medical History encounters,Able to view NCD Medical History encounters,,
 14e7490d-d6f2-44c1-a8a2-ac57e44981ff,Edit NCD Medical History encounters,Able to edit NCD Medical History encounters,,
 acd13a9a-6df3-4881-8a61-16f6a28ba1d3,View NCD Consultation encounters,Able to view NCD Consultation encounters,,
-24801d10-fd08-44d6-8b63-b649ff28a672,Edit NCD Consultation encounters,Able to edit NCD Consultation encounters,,
\ No newline at end of file
+24801d10-fd08-44d6-8b63-b649ff28a672,Edit NCD Consultation encounters,Able to edit NCD Consultation encounters,,
+a626f0bc-f4d7-4c5d-b2e3-afb34cb2d180,View CC Screening encounters,Able to view CC Screening encounters,,
```

Comment:
It would be better we use the actual name of the encounter
```code
View Cervical Cancer Screening encounters
```

Reply:
This has been addressed [here](https://github.com/openmrs/ozone-distro-cambodia/pull/233/commits/27a281a7abf097eca8fa1277433e642d16a42b9f)